libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/Matrix/MatrixUtils.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export type IMatrixSingleCategory =
+  | IMatrixSingleClassificationCategory
+  | IMatrixSingleRegressionCategory;
+
+export type IMatrixSingleClassificationCategory = {
+  value: number;
+  intervalMax?: never;
+  intervalMin?: never;
+};
+export type IMatrixSingleRegressionCategory = {
+  value: number;
+  intervalMax: number;
+  intervalMin: number;
+};
+
+export function formatCategory(
```

Comment:
this looks like a display process function, extract as a react component could be better than define a ts function.

Reply:
done, made formatCategory a class/tsx file